### DISCUSSION TITLE
How to create or insert sortable table

### DISCUSSION THREAD

**Discussion by kdssoftware** [PHPOffice/PhpSpreadsheet#2337] [Q&A] [answered]
I've created a .xlsx file using phpspreadsheet with lots of data and columns.
Is there a way to add or set the data in a sortable (filterable) table.
I'm looking for the equivalent of Excel's Create Table function. (which can be found in Insert > Table).
![image](https://user-images.githubusercontent.com/10829524/137497390-8a46877e-4db8-4255-a273-d1587d0cd2a9.png)

**Comment by MarkBaker** [comment, marked as answer]
You need to specify an autoFilter:
```
$spreadsheet->getActiveSheet()->setAutoFilter('A1:F42');
```
This is all [fully documented](https://phpspreadsheet.readthedocs.io/en/latest/topics/autofilters/)

**Reply by kdssoftware** [reply]
Works like a charm, thanks for the response !
